File: src/main/java/com/lifetech/application/dto/PersonProfileDTO.java
package com.lifetech.application.dto;

public class PersonProfileDTO extends PersistableElementDTO {

    //private Long id;
    private String nom;
    private String prenom;
    private int age;
    private String adress;
    private int postal_code;
    private String adress_email;
    private int tel_number;
    private String chronic_disease;
    private int estimated_mobility;
    private boolean smoking;
    private float smoking_time;
    private boolean drinker;
    private boolean medical_emergencies;
    private String hobbies;
    private float revenue;
    private String profile;
    private int priority_coef;

     /*@Override
   public Long getId() {
        return id;
    }

    @Override
    public void setId(Long id) {
        this.id = id;
    }
*/
    public PersonProfileDTO(){}

    public PersonProfileDTO(String nom, String prenom, int age, String adress, int postal_code, String adress_email, int tel_number, String chronic_disease, int estimated_mobility, boolean smoking, float smoking_time, boolean drinker, boolean medical_emergencies, String hobbies, float revenue, String profile, int priority_coef) {
        this.nom = nom;
        this.prenom = prenom;
        this.age = age;
        this.adress = adress;
        this.postal_code = postal_code;
        this.adress_email = adress_email;
        this.tel_number = tel_number;
        this.chronic_disease = chronic_disease;
        this.estimated_mobility = estimated_mobility;
        this.smoking = smoking;
        this.smoking_time = smoking_time;
        this.drinker = drinker;
        this.medical_emergencies = medical_emergencies;
        this.hobbies = hobbies;
        this.revenue = revenue;
        this.profile = profile;
        this.priority_coef = priority_coef;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public String getPrenom() {
        return prenom;
    }

    public void setPrenom(String prenom) {
        this.prenom = prenom;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }

    public String getAdress() {
        return adress;
    }

    public void setAdress(String adress) {
        this.adress = adress;
    }

    public int getPostal_code() {
        return postal_code;
    }

    public void setPostal_code(int postal_code) {
        this.postal_code = postal_code;
    }

    public String getAdress_email() {
        return adress_email;
    }

    public void setAdress_email(String adress_email) {
        this.adress_email = adress_email;
    }

    public int getTel_number() {
        return tel_number;
    }

    public void setTel_number(int tel_number) {
        this.tel_number = tel_number;
    }

    public String getChronic_disease() {
        return chronic_disease;
    }

    public void setChronic_disease(String chronic_disease) {
        this.chronic_disease = chronic_disease;
    }

    public int getEstimated_mobility() {
        return estimated_mobility;
    }

    public void setEstimated_mobility(int estimated_mobility) {
        this.estimated_mobility = estimated_mobility;
    }

    public boolean isSmoking() {
        return smoking;
    }

    public void setSmoking(boolean smoking) {
        this.smoking = smoking;
    }

    public float getSmoking_time() {
        return smoking_time;
    }

    public void setSmoking_time(float smoking_time) {
        this.smoking_time = smoking_time;
    }

    public boolean isDrinker() {
        return drinker;
    }

    public void setDrinker(boolean drinker) {
        this.drinker = drinker;
    }

    public boolean isMedical_emergencies() {
        return medical_emergencies;
    }

    public void setMedical_emergencies(boolean medical_emergencies) {
        this.medical_emergencies = medical_emergencies;
    }

    public String getHobbies() {
        return hobbies;
    }

    public void setHobbies(String hobbies) {
        this.hobbies = hobbies;
    }

    public float getRevenue() {
        return revenue;
    }

    public void setRevenue(float revenue) {
        this.revenue = revenue;
    }

    public String getProfile() {
        return profile;
    }

    public void setProfile(String profile) {
        this.profile = profile;
    }

    public int getPriority_coef() {
        return priority_coef;
    }

    public void setPriority_coef(int priority_coef) {
        this.priority_coef = priority_coef;
    }
}

File: src/main/demo-ui/src/app/dto/PersonProfileDTO.ts

export class PersonProfileDTO{


  private _id: number;
  private _nom: string;
  private _prenom: string;
  private _age: bigint;
  private _adress: string;
  private _postal_code: bigint;
  private _adress_email: string;
  private _tel_number: bigint;
  private _chronic_disease: string;
  private _estimated_mobility: bigint;
  private _smoking: boolean;
  private _smoking_time: number;
  private _drinker: boolean;
  private _medical_emergencies: boolean;
  private _hobbies: string;
  private _revenue: number;
  private _profile: string;
  private _priority_coef: number;


  get id(): number {
    return this._id;
  }

  set id(value: number) {
    this._id = value;
  }

  get nom(): string {
    return this._nom;
  }

  set nom(value: string) {
    this._nom = value;
  }

  get prenom(): string {
    return this._prenom;
  }

  set prenom(value: string) {
    this._prenom = value;
  }

  get age(): bigint {
    return this._age;
  }

  set age(value: bigint) {
    this._age = value;
  }

  get adress(): string {
    return this._adress;
  }

  set adress(value: string) {
    this._adress = value;
  }

  get postal_code(): bigint {
    return this._postal_code;
  }

  set postal_code(value: bigint) {
    this._postal_code = value;
  }

  get adress_email(): string {
    return this._adress_email;
  }

  set adress_email(value: string) {
    this._adress_email = value;
  }

  get tel_number(): bigint {
    return this._tel_number;
  }

  set tel_number(value: bigint) {
    this._tel_number = value;
  }

  get chronic_disease(): string {
    return this._chronic_disease;
  }

  set chronic_disease(value: string) {
    this._chronic_disease = value;
  }

  get estimated_mobility(): bigint {
    return this._estimated_mobility;
  }

  set estimated_mobility(value: bigint) {
    this._estimated_mobility = value;
  }

  get smoking(): boolean {
    return this._smoking;
  }

  set smoking(value: boolean) {
    this._smoking = value;
  }

  get smoking_time(): number {
    return this._smoking_time;
  }

  set smoking_time(value: number) {
    this._smoking_time = value;
  }

  get drinker(): boolean {
    return this._drinker;
  }

  set drinker(value: boolean) {
    this._drinker = value;
  }

  get medical_emergencies(): boolean {
    return this._medical_emergencies;
  }

  set medical_emergencies(value: boolean) {
    this._medical_emergencies = value;
  }

  get hobbies(): string {
    return this._hobbies;
  }

  set hobbies(value: string) {
    this._hobbies = value;
  }

  get revenue(): number {
    return this._revenue;
  }

  set revenue(value: number) {
    this._revenue = value;
  }

  get profile(): string {
    return this._profile;
  }

  set profile(value: string) {
    this._profile = value;
  }

  get priority_coef(): number {
    return this._priority_coef;
  }

  set priority_coef(value: number) {
    this._priority_coef = value;
  }
}
